scripts/fixtures_qa/one_type.json

```diff
@@ -0,0 +1,119 @@
+[
+        {
+            "mimetype": 50,
+            "type": {
+                "name": "Generic",
+                "major_version": 1, 
+                "minor_version": 0, 
+                "patch_version": 0
+            },
+            "data": {
+                "price": 23.09, "symbol": "AMZN", "timestamp": 1690928634000, "volume": 300
+            }
```

Comment:
Let's also add a `meta` field with key-value strings. They don't have to be the same for each event, but maybe you can do something like:

```
"meta": {
     "symbol": "AMZN",
}
```

Just so we have something to display for metadata on the frontend side.